rest-nvd/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+
+services:
+  rest-nvd:
+    build: .
+    ports:
+      - "22222:80"
```

Comment:
Hello @copernico,

In my opinion, you shouldn't instruct Gunicorn to use HTTPS at all, you should instead disable it if it's now enabled. In the end, the `nvd-rest` container should only `expose` the port 80 to the local Docker network and it will be only queried by HAProxy and by the two backends.

So HAProxy will do the HTTPS termination. I also don't see any Poodle/Beast risks as the container shouldn't be exposed directly to the Internet. Afterall we use HAProxy as a load balancer but mainly as an ingress.

So, in my opinion, you should just remove this `ports: - "22222:80"` and instead `expose` the 80. 

But maybe for some specific reason, you want to allow `nvd-rest` to be queried directly from the Internet, if so what I just said doesn't apply.